src/uct/sm/self/self_ep.c

```diff
@@ -118,12 +120,14 @@ ssize_t uct_self_ep_am_bcopy(uct_ep_h tl_ep, uint8_t id,
     /* Receive part */
     uct_iface_trace_am(&self_iface->super, UCT_AM_TRACE_TYPE_RECV, id, payload,
                        length, "RX: AM_BCOPY");
-    status = uct_iface_invoke_am(&self_iface->super, id, payload, length, desc);
+    status = uct_iface_invoke_am(&self_iface->super, id, payload, length,
```

Comment:
I'm not sure because pack/unpack callbacks for am_bcopy may assume different data layout, the same regarding to header + payload for am_short